docker/update-binaries.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# SPDX-FileCopyrightText: 2019 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+bot_name="CI bot"
+branch="update-binaries-list-$BUILDKITE_TAG"
+
+git config --local user.email "hi@serokell.io"
+git config --local user.name "$bot_name"
+git fetch --all
+git checkout -B "$branch"
+
+python3 package/scripts/update-binaries-list.py
+
+git add --all
+git commit -m "Updated binaries for $BUILDKITE_TAG release" --gpg-sign="tezos-packaging@serokell.io"
```

Comment:
For some reason [this commit](https://github.com/serokell/tezos-packaging/pull/752/commits/5d02cf1840a47af892a73d795e9b970dbd4a5367) is marked as `Unverified` which means that it's unsigned 🤔